docs/developer/getting-started/create-edit-forms.md

```diff
@@ -0,0 +1,101 @@
+# Create/Edit Forms
+
+# Standard Create/Edit Form Architecture
+
+The reusable CruResource component and its mixins were created to solve a problem particular to Kubernetes resource management. In older Rancher versions, Rancher sometimes restricted fields from being edited or added form validation in Norman, the backend API that predated Steve. Originally, Norman was designed to restrict users from editing resources in the Kubernetes cluster directly so that Norman could control how resources were configured.
+
+But in modern DevOps, many users want the ability to automate their workflows with GitOps, which usually requires tracking all of the YAML files for Kubernetes resources in a Git repository, and also requires users to be able to edit resources through config files instead of only manipulating them through the UI.
+
+The problem was that when Rancher exposed access to Kubernetes clusters directly, users could not only access Kubernetes resources through Norman, but they could also edit the resource’s YAML config files and bypass the restrictions that Norman could impose.
+
+Rancher’s reusable forms are intended to be flexible enough to allow YAML files to be created and edited through the Rancher UI, while also providing form validation for those files at the same time. The reusability also allows Rancher to quickly adapt to new Kubernetes custom resources that are introduced for Helm chart apps that are not primarily developed in-house, such as the monitoring and logging applications.
+
+# Creating a New Create/Edit Form for a Resource
+
+This section describes the workflow for creating forms to create or edit resources in Rancher. We recommend using the CruResource component for create/edit forms because it is designed to handle resource that can also be managed through YAML config files, with process automated through GitOps.
+
+You will need to create a file for the create/edit form. For normal Kubernetes resources, this file should be located in the `@/components/edit` directory. For projects maintained in non-Rancher repositories, all files related to that project should be kept in a product-specific folder in `/products`.
+
+The file will need these imports:
+```
+import CreateEditView from '@/mixins/create-edit-view/impl';
+// An example types file 
+import { EPINIO_TYPES } from '@/products/epinio/types';
```

Comment:
For a basic example best not to include epinio which might one day live in another repo

Reply:
changed to `import { POD } from '@/config/types';`